skycoin/skywire
# Limit the Skychay UI to localhost

Changes:	
- Now the SkychatUI can only been opened from localhost or 127.0.0.1.

How to test this PR:
Recompile the UI and use it to open the Skychat UI. It will open it from 127.0.0.1.

Note: should this behaviour be configurable?
